pom.xml

```diff
@@ -1,26 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.example</groupId>
+	<groupId>org.reactome.release</groupId>
 	<artifactId>release-external-dependencies</artifactId>
 	<version>1.0-SNAPSHOT</version>
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<configuration>
-					<source>8</source>
-					<target>8</target>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+	<packaging>jar</packaging>
+
+	<name>release-external-dependencies</name>
+	<description>
+		A program to check third-party resources required for the Reactome data release process.  Produces output to
+		inform the user if which resource(s) are available and correct and which are not.
+	</description>
+	<url>https://reactome.org</url>
+	<licenses>
+		<license>
+			<name>Apache License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.html</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<name>Joel Weiser</name>
+			<email>joel.weiser@oicr.on.ca</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>
+			scm:git:git://github.com/reactome/release-external-dependencies.git
+		</connection>
+		<developerConnection>
+			scm:git:ssh://github.com/reactome/release-external-dependencies.git
+		</developerConnection>
+		<url>https://github.com/reactome/release-external-dependencies.git</url>
+		<tag>develop</tag>
+	</scm>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<!--
+		integration tests run by default when running the command "mvn verify"; can be overridden using the option
+		"-DskipITs=true"
+		-->
+		<skipITs>false</skipITs>
```

Comment:
Yes, that's true but changing the default value in the file changes the configuration in a more permanent way.  I set it to skipping ITs as false, because I think we would like to run integration tests by default, but the command-line option gives a convenient way to override that value on a one-time basis without affecting the default behaviour of running all tests (integration and unit tests).